examples/VB_ASR/helpers.py

```diff
@@ -0,0 +1,217 @@
+import numpy as np
```

Comment:
helper is not very intuitive, in addition, these files should go into the digital twin section, since those are not the example, but general functions that could be used in multiple examples.

Reply:
I'll move this scriptto the digitaltwin library and rename it to helper_plots.py or helper_plot_distributions.py (or maybe the latter without helper).